DotNet/AdventOfCode2022/Day03RucksackReorganization/Domain/Group.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Day03RucksackReorganization.Services;
+
+namespace Day03RucksackReorganization.Domain
+{
+    public class Group
+    {
+        public Rucksack FirstRucksack { get; private set; }
+        public Rucksack SecondRucksack { get; private set; }
+        public Rucksack ThirdRucksack { get; private set; }
+
+        public Group(Rucksack firstRucksack, Rucksack secondRucksack, Rucksack thirdRucksack)
+        {
+            FirstRucksack = firstRucksack;
+            SecondRucksack = secondRucksack;
+            ThirdRucksack = thirdRucksack;
+        }

```

Comment:
It isn't, since this is a solution for the specific use case, part 2. If instead a custom amount of rucksacks is required, then a collection should be used where rucksacks can be added.